src/xhr.js

```diff
@@ -65,10 +65,17 @@ var XHR = {
                 onreadystatechange = undefined;
 
                 // TODO(cory): have the notifier log an internal error on non-200 response codes
+                var jsonResponse = RollbarJSON.parse(request.responseText);
                 if (request.status === 200) {
-                  callback(null, RollbarJSON.parse(request.responseText));
+                  callback(null, jsonResponse);
                 } else if (Util.isType(request.status, 'number') &&
-                            request.status >= 400 && request.status < 600) {
+                  request.status >= 400 && request.status < 600) {
+
+                  if (request.status == 403 && console) {
```

Comment:
There are some other places that we use `console` directly; I'm guessing it may work because of this polyfill: https://github.com/rollbar/rollbar.js/blob/master/src/util.js#L1 . Perhaps we should change those all to use a wrapper function that we define in util.js?